decidim-comments/app/frontend/comments/comment.component.jsx

```diff
@@ -0,0 +1,43 @@
+import { propType }    from 'graphql-anywhere';
+import gql             from 'graphql-tag';
+
+import commentFragment from './comment.fragment.graphql'
+
+/**
+ * @returns {ReactComponent} - A single comment component with the author info and the comment's body
+ */
+const Comment = ({ comment: { author, body, createdAt } }) => (
+  <article className="comment">
+    <div className="comment__header">
+      <div className="author-data">
+        <div className="author-data__main">
+          <div className="author author--inline">
+            <a className="author__avatar">
+              <img alt="avatar" src="../../assets/images/demo-avatar.jpg" />
+            </a>
+            <a className="author__name">{author.name}</a>
+            <time dateTime={createdAt}>{` ${createdAt}`}</time>
```

Comment:
It's a design issue here because we are missing some kind of margin. I'm not sure about the localisation neither, is it something we should handle in the backend?